frontend/www/js/omegaup/components/arena/Runs.vue

```diff
@@ -219,14 +226,18 @@
                 <tt>{{ run.guid.substring(0, 8) }}</tt>
               </acronym>
             </td>
-            <td v-if="showUser" class="text-break-all">
+            <td
+              v-if="showUser"
+              class="text-break-all"
+              :data-username="run.username"
+            >
               <omegaup-user-username
                 :classname="run.classname"
                 :username="run.username"
                 :country="run.country_id"
                 :linkify="true"
                 :emit-click-event="true"
-                @click="(username) => (filterUsername = username)"
+                @click="(user) => (filterUser = { key: user, value: user })"
```

Comment:
```suggestion
                @click="(username) => (filterUsername = { key: user, value: username })"
```

acá lo que se está pasando es un username, no un usuario.

Reply:
Listo